<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Print preview</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox_pupil" >
       <property name="title" >
        <string>Pupil</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>3</number>
        </property>
        <property name="leftMargin" >
         <number>3</number>
        </property>
        <property name="topMargin" >
         <number>3</number>
        </property>
        <property name="rightMargin" >
         <number>3</number>
        </property>
        <property name="bottomMargin" >
         <number>3</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBox_pupil" >
          <property name="windowModality" >
           <enum>Qt::WindowModal</enum>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_page" >
       <property name="title" >
        <string>Page</string>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QComboBox" name="comboBox_page" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QRadioButton" name="radioButton_l" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_r" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>right</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton" >
           <property name="text" >
            <string>Done</string>
           </property>
           <property name="icon" >
            <iconset>icons/quit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_size" >
         <property name="title" >
          <string>size</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QSlider" name="sizeSlider" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="MinimumExpanding" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum" >
             <number>20</number>
            </property>
            <property name="pageStep" >
             <number>1</number>
            </property>
            <property name="sliderPosition" >
             <number>10</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
